lib/representable/hash_methods.rb

```diff
@@ -21,7 +21,15 @@ def filter_keys_for!(hash, options)
       excluding = options[:exclude]
       # TODO: use same filtering method as in normal representer in Representable#create_representation_with.
       return hash unless props = options.delete(:exclude) || options.delete(:include)
-      hash.reject { |k,v| excluding ? props.include?(k.to_sym) : !props.include?(k.to_sym) }
+
+      # Converting to Array and back to Hash
+      # just to avoid a `warning: extra states are no longer copied:`
+      # See: https://bugs.ruby-lang.org/issues/9275
+      ary = hash.to_a.reject do |k,v|
```

Comment:
Hi @myabc, this ```Hash#reject``` warning is so... I don't have a word. 😞

I'm converting to an Array here just to __drop__ these __extra__ state of the instance.
What I wanna say is, I don't really need **the conversion** itself.
I need a way to "drop" any state and give a clear Hash (or Array) to #reject so it doesn't issue the warning.
I couldn't come out with another way to fix this.

Transproc may probably fall into the performance problem @apotonick said. But, I should benchmark them to see.